dkg/dkg.go

```diff
@@ -26,30 +26,28 @@ type Actor interface {
 
 	// Encrypt encrypts the given message into kyber points
 	// using the DKG public key
-	Encrypt(message []byte) (K, C kyber.Point, remainder []byte, err error)
-	// Decrypt decrypts a pair of kyber points into the original message
-	// using the DKG internal private key
-	Decrypt(K, C kyber.Point) ([]byte, error)
+	// where K is the ephemeral DH (Diffie-Hellman) public key
```

Comment:
I'm a bit confused.
Is this `K` the same as the renamed `U`, that referred to randomness ? 
Or is randomness here supposed to be the DH public key ?

Reply:
As far as I understand it, it's all the same. This comment comes from cothorithy. I thought it would be good to have this info here.